triplea-game/triplea
# GitHub map download

## Functional Changes
- do not prompt the user for the map list XML URL, go directly to the map download screen
- store the location of the map list XML in a property file 'mapDownload.properties'
- add a triplea_maps.xml, update it so it has URLs for the github maps, and clean it up

## Testing
- Check out the branch:
  - git remote add github.com:DanVanAtta/triplea.git DanVanAtta
  - git checkout DanVanAtta github_map_download

- To test downloading maps from github and the updated map download UX:
  - Update the file `mapDownload.properties`, uncomment the local `triplea_maps.xml` configuration line
  - launch the game
  - download a map 


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/triplea-game/triplea/286)
